client/signup/steps/design-type-with-store/index.jsx

```diff
@@ -117,19 +117,14 @@ class DesignTypeWithStoreStep extends Component {
 
 	renderChoice = choice => {
 		return (
-			<Card className="design-type-with-store__choice" key={ choice.type }>
-				<a
-					className="design-type-with-store__choice-link"
-					href="#"
-					onClick={ this.handleChoiceClick( choice.type ) }
-				>
-					<div className="design-type-with-store__image">{ choice.image }</div>
-					<div className="design-type-with-store__choice-copy">
-						<span className="button is-compact design-type-with-store__cta">{ choice.label }</span>
-						<p className="design-type-with-store__choice-description">{ choice.description }</p>
-					</div>
-				</a>
-			</Card>
+			<Tile
+				buttonLabel={ choice.label }
+				description={ choice.description }
+				href="#"
```

Comment:
I believe this `href` is redundant and has the negative side effect of turning a `<div>` into an `<a href="#">`.

There is a `button` which handles focus nicely and keyboard activation nicely.
There is a `cursor:pointer` style on the `card.is-clickable`.

Reply:
That's a good point 👍 I was planning to have this just because it was there in the original version, but it doesn't hurt to improve and remove this in this PR. Thanks 👍 

Removed in https://github.com/Automattic/wp-calypso/pull/24979/commits/a6b88c4afdd2de215903dd7c1b3d6128f40857b8 🔪